src/routes/term.rs

```diff
@@ -17,6 +17,6 @@ pub async fn term(
     let courses = state.courses(term)?;
 
     Ok(components::base(html! {
-        (components::container::calendar_container(term, &courses))
+        (components::container::calendar_container(term, &courses, &[]))
```

Comment:
i could have added a TODO here to implement the passing of course information to `calendar_container`

Reply:
Yeah that seems like a good idea, might as well.